Kong/docs.konghq.com
# Move terraform aws module from plugin hub to install options

### Description

Turning this page: https://docs.konghq.com/hub/kong-inc/kong-terraform-aws/
Into an install option on the Kong Gateway installation overview page. 

This is the only Kong-owned, not third-party, plugin hub entry that is getting cleaned out. 

Reasons:
* This is not a plugin
* Kong actively maintains this repo, but we have no other docs on it
* Unlike the Bitnami deployment listing, this is in our own control, so we should list it in our own docs

### Testing instructions

Netlify link: https://deploy-preview-5497--kongdocs.netlify.app/gateway/latest/install/

https://deploy-preview-5497--kongdocs.netlify.app/hub/kong-inc/kong-terraform-aws/ should redirect to https://github.com/kong/kong-terraform-aws

### Checklist
